BattleshipFrontend/src/app/api/models.ts

```diff
@@ -1,78 +1,70 @@
-export interface PlayerFront {
+export interface PlayerInitialData {
   name: string;
   team: number;
 }
 
+export interface DragModel {
+  cellX: number;
+  cellY: number;
+  type: string;
+  row: number;
+  col: number;
+}
+
+export interface ShipData {
+  name: string;
+  length: number;
+  coordinates: Coordinates[];
+}
+
+export interface Coordinates {
+  x: number;
+  y: number;
+}
+
+export interface PlayerShipsData {
+  playerId: string;
```

Comment:
this is a number not a string, we solve later with merge of models. but when u create a new one might be some problems